tests.py

```diff
@@ -91,21 +76,26 @@ def test_delete_animal_from_cage(self):
 
 
 class TestPredatorEatsPrey(unittest.TestCase):
-    """Creates and adds two animals to a cage, one of them it's a predator and the other one is a prey.
+    """Creates and adds two animals to a cage,
+    one of them it's a predator and the other one is a prey.
     Expects the number of animals in the cage to be 1.
-    This is tested in two different ways, first adding a predator and then a prey. And secondly adding the prey first.
+    This is tested in two different ways,
+    first adding a predator and then a prey.
+    And secondly adding the prey first.
     Both behaviours should result in the predator being left on its own.
     """
 
-    def test_predator_eats_new_prey(self):
+    @patch("builtins.print")
+    def test_predator_eats_new_prey(self, mock_print):
```

Comment:
See if you can work out how to also test that when the animal gets eaten, the correct thing gets printed.  This goes for both methods in this class.